src/elements/content-preview/ContentPreview.js

```diff
@@ -792,7 +792,7 @@ class ContentPreview extends React.PureComponent<Props, State> {
         this.preview.addListener('thumbnailsClose', () => this.setState({ isThumbnailSidebarOpen: false }));
 
         if (showAnnotationsControls) {
-            this.preview.addListener('annotator', onAnnotator);
+            this.preview.addListener('annotator_create', onAnnotator);
```

Comment:
Can you remind me of the reason for this change? Does the annotator `annotations_fetch_error` event fire before preview's `annotator` event?

Reply:
Yeah, `annotations_fetch_error` event fires before `annotator` event.